### PR TITLE
Remove unnecessary speed and concurrent limitation

### PR DESCRIPTION
**Pull request by fengou1**
### What problem does this PR solve? 
remove unnecessary limitation

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Release note

 -

### CODE REVIEW

**Comment by glorv** [comment]
BTW, do we have a plan to make `--rate-limit compatible with `--concurrency` 🤔️

File: tests/br_incremental_index/run.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ run_sql "ALTER TABLE ${DB}.${TABLE} DROP COLUMN c3;";
 # incremental backup
 echo "incremental backup start..."
 last_backup_ts=$(run_br validate decode --field="end-version" -s "local://$TEST_DIR/$DB/full" | grep -oE "^[0-9]+")
-run_br --pd $PD_ADDR backup table -s "local://$TEST_DIR/$DB/inc" --db $DB -t $TABLE --ratelimit 5 --concurrency 4 --lastbackupts $last_backup_ts
+run_br --pd $PD_ADDR backup table -s "local://$TEST_DIR/$DB/inc" --db $DB -t $TABLE ---lastbackupts $last_backup_ts
```
Comment (kennytm) [inline tests/br_incremental_index/run.sh:45]:
```suggestion
run_br --pd $PD_ADDR backup table -s "local://$TEST_DIR/$DB/inc" --db $DB -t $TABLE --lastbackupts $last_backup_ts
```

```
[2021-07-12T03:35:01.543Z] [2021/07/12 11:35:01.349 +08:00] [ERROR] [main.go:58] ["br failed"] [error="bad flag syntax: ---lastbackupts"] [stack="github.com/pingcap/br/cmd/br.main\n\t/home/jenkins/agent/workspace/br_ghpr_unit_and_integration_test/go/src/github.com/pingcap/br/cmd/br/main.go:58\ngithub.com/pingcap/br/cmd/br.TestRunMain.func1\n\t/home/jenkins/agent/workspace/br_ghpr_unit_and_integration_test/go/src/github.com/pingcap/br/cmd/br/main_test.go:39"]
```

**Review by kennytm — APPROVED** [review]

**Comment by fengou1** [comment]
> BTW, do we have a plan to make `--rate-limit compatible with `--concurrency` 🤔️

This is good question, I can not answer at this moment, will keep it in mind and continues to study it.

**Review by Leavrth — APPROVED** [review]

File: tests/br_s3/run.sh [hunk 2]
```diff
@@ -111,7 +110,6 @@ for p in $(seq 2); do
   rm -f $RESTORE_LOG
   unset BR_LOG_TO_TERM
   ( run_br restore full -s "s3://mybucket/$DB?$S3_KEY" --pd $PD_ADDR --s3.endpoint="http://$S3_ENDPOINT" \
-      --ratelimit 1 \
```
Comment (YuJuncen) [inline tests/br_s3/run.sh:114]: I think maybe we need some mechanism to sync the state after removing this...

At L117-120, we sleep 3s for waiting for BR enter the phase of 'download data from external storage', then we trigger a S3 outage. Without this rate limit, the restoration may finish in 3s. And then this test case become no-meaning.

A simple mechanism to notify could be a signal file created by failpoint. (See the `s3-outage-during-writing-file`).

Comment (fengou1) [reply tests/br_s3/run.sh:114]: chatted with Juncen, this rate limit matter to the test case, it shall be reverted.

**Pull request merged**